markdown_it/common/utils.py

```diff
@@ -91,25 +91,24 @@ def fromCodePoint(c: int) -> str:
     r'\\([!"#$%&\'()*+,\-.\/:;<=>?@[\\\]^_`{|}~])' + "|" + r"&([a-z#][a-z0-9]{1,31});",
     re.IGNORECASE,
 )
-DIGITAL_ENTITY_BASE10_RE = re.compile(r"#([0-9]{1,8})")
-DIGITAL_ENTITY_BASE16_RE = re.compile(r"#x([a-f0-9]{1,8})", re.IGNORECASE)
+DIGITAL_ENTITY_TEST_RE = re.compile(
```

Comment:
This change seems separate to the PR title?
Can you split it into a separate PR

Reply:
Most changes here are unrelated refactors actually. This is a port of the diff in the first post. Should something else be split too?

Reply:
Yeh ideally it should be split up fully into "logical changes" 😅

Reply:
:+1: Pushed a new commit history. Would this split work?

Reply:
> Would this split work?

Please split them up into separate PRs, with a clear description of their intention; the smaller the PR better, and the more likely I can just merge them without much effort/delay